File: problems/problem_7/fullboilerplate.java
import java.util.*;
public class Main {
    // {{USER_CODE}}
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String haystack = sc.hasNext() ? sc.next() : "";
        String needle = sc.hasNext() ? sc.next() : "";
        System.out.println(strStr(haystack, needle));
    }
}

File: problems/problem_7/fullboilerplate.py
import sys
# {{USER_CODE}}
if __name__ == "__main__":
    inp = sys.stdin.read().split()
    haystack = inp[0] if len(inp) > 0 else ""
    needle = inp[1] if len(inp) > 1 else ""
    print(strStr(haystack, needle))
